packages/react-make-styles/src/makeStyles.ts

```diff
@@ -18,10 +18,6 @@ function useRenderer(document: Document | undefined): MakeStylesRenderer {
 export function makeStyles<Slots extends string>(stylesBySlots: Record<Slots, MakeStylesStyleRule<Theme>>) {
   const getStyles = vanillaMakeStyles(stylesBySlots);
 
-  if (process.env.NODE_ENV === 'test') {
-    return () => ({} as Record<Slots, string>);
```

Comment:
How will this serializer work for unit tests that require styles with theme values ?

Reply:
Well, might be a good idea to add a test for that. but techinically should ismply work.
I'll add theme values on tests just to ensure

Reply:
And to unit test styles with theme values an actual theme will be required with `FluentProvider` or `ThemeProvider` right ?

Reply:
Yes. at the moment it is required to add a provider. perhaps we can manage to add a better solution? @layershifter

Reply:
I really would like to not require ThemeProvider to unit test components. Apart forcing users to mock them or applying a theme mock ourselves I can't think of anything else 

@layershifter @miroslavstastny any ideas ?

Reply:
If the component has a dependency on ThemeProvider to work, it makes sense to mock the provider to give you the values, or am I misinterpreting the problem?

Reply:
Yeah, one possible idea would be to implement a Proxy to mock the behaviour of the Theme, without creating an unecessary dependency. I'm considering this as a possible solution @andrefcdias